app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::ItemsController < ApiController
+  before_action :authenticate_user
+
+  def create
+    list = List.find(params[:list_id])
```

Comment:
You might want to scope this by the current user to avoid them being able to add items to other people's lists.

You'll need to change your `authenticate_user` method to store the result of the authentication in a `@current_user` variable and then change this line to `list = @current_user.lists.find(params[:list_id])`.